MetaMorpheus/GuiFunctions/MetaDraw/MetaDrawLogic.cs

```diff
@@ -1056,6 +1059,10 @@ private void LoadSpectraFiles(out List<string> errors)
 
                     var spectraFile = MsDataFileReader.GetDataFile(filepath);
                     spectraFile.InitiateDynamicConnection();
+                    if (spectraFile is TimsTofFileReader timsTofDataFile)
```

Comment:
Will this cause initiate dynamic connection to be called twice? Is that an issue?